src/extensions/default/JavaScriptCodeHints/tern-worker.js

```diff
@@ -587,14 +588,24 @@ var config = {};
              * @param {string} path     - the path of the file
              */
             function handlePrimePump(path) {
-                var fileInfo = createEmptyUpdate(path),
+                var fileName = path;
+                if (ternServer.projectDir && fileName.indexOf(ternServer.projectDir) === 0) {
```

Comment:
if you don't mind one last change, for future readability, I'd extract this to `_getDenormalizedFilename` as this basically reverts `_getNormalizedFilename` if I'm not mistaken